Weekly cronbuild failing due to WIX upgrade breaking Windows build

The weekly cronbuild started failing 3/8/24:
```
  dir "c:\Program Files (x86)\WiX Toolset"*
03/04/2024  10:37 PM    <DIR>          WiX Toolset v3.14
  set PATH=C:\Program Files (x86)\WiX Toolset v3.11\bin;%PATH%

-- DRMF version number: 1.0.19760
-- NSIS found: will build NSIS-based installer
CMake Error at CMakeLists.txt:2300 (message):
  Installed NSIS has a too-small path limit.  See i#1029.
```

Prior week's 3/1/24 which passed:
```
  dir "c:\Program Files (x86)\WiX Toolset"*
02/25/2024  10:29 PM    <DIR>          WiX Toolset v3.11
  set PATH=C:\Program Files (x86)\WiX Toolset v3.11\bin;%PATH%

-- DRMF version number: 1.0.19760
-- WIX found: will build WIX-based .msi installer
```

Looks like we hardcode 3.11 in the yml file.

--- a/.github/workflows/ci-package.yml
+++ b/.github/workflows/ci-package.yml
@@ -471,7 +471,7 @@ jobs:
         7z x c:\projects\install\doxygen.zip -oc:\projects\install\doxygen > nul
         set PATH=c:\projects\install\doxygen;%PATH%
         dir "c:\Program Files (x86)\WiX Toolset"*
-        set PATH=C:\Program Files (x86)\WiX Toolset v3.11\bin;%PATH%
+        set PATH=C:\Program Files (x86)\WiX Toolset v3.14\bin;%PATH%
         call "C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Auxiliary/Build/vcvars32.bat"
         echo ------ Running suite ------
         echo PATH is "%PATH%"

